docs/topics/scale.md

```diff
@@ -10,7 +10,7 @@ This guide assumes you already have deployed a cluster using aks-engine. For mor
 
 The `aks-engine scale` command can increase or decrease the number of nodes in an existing agent pool in an `aks-engine` Kubernetes cluster. Nodes will always be added or removed from the end of the agent pool. Nodes will be cordoned and drained before deletion.
 
-This guide will assume you have a cluster deployed and the output for the deployed cluster is stored at _output/mycluster. It will also assume there is a node pool named "agentpool1" in your cluster. AKS Engine will default to storing the output at ./_output/<dnsPrefix> from where the aks-engine command was run.
+This guide will assume you have a cluster deployed and the apimodel originally used to deploy that cluster is stored at ./apimodel.json. It will also assume there is a node pool named "agentpool1" in your cluster.
```

Comment:
should we use `_output/<cluster-name>/apimodel.json` as the example apimodel filepath since that's the default location deploy/generate use?

Reply:
or `_output/<dnsPrefix>/apimodel.json`

Reply:
Will do